demisto_sdk/commands/common/errors.py

```diff
@@ -988,6 +990,16 @@ def invalid_hidden_attribute_for_param(param_name: str, invalid_value: str):
     def invalid_deprecated_integration_display_name():
         return 'The display_name (display) of all deprecated integrations should end with (Deprecated)".'
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_integration_deprecation__only_display_name_suffix():
+        return 'All integrations whose display_names end with `(Deprecated)` must have `deprecated:true`.'
```

Comment:
Add a suggestion to run `format` with the flag `-d, --deprecate` to fully deprecate an content item

Reply:
done, thanks